Windows AMI system check failures outage

ccooper opened the pull request:
It's my first attempt at a Taskcluster retrospective. Be gentle!

ccooper committed:
Add new retrospective for Windows AMI system check failures on
2018-02-13

ccooper committed:
Update Timeline, Fallout, and Action Items

ccooper committed:
Fill out fix section

ccooper committed:
Stub out remediation section

ccooper committed:
Flesh out Remediation section

ccooper committed:
Add link to provisioner documentation bug

imbstack reviewed (approved):
This looks great to me! The only nit is that I think we generally try to make these written from the perspective of the team at large so anywhere that is "I" is generally a "we". Otherwise this looks great.

ccooper committed:
Replace "I" with "we"

djmitche reviewed (approved):
Looks good!

It might be nice to use full names instead of lower-cased nicknames where specific people are noted.

djmitche commented:
@ccooper I think you can merge this :)

ccooper committed:
Update 2018-02-13-windows-system-check-failures.md

Changed irc nicks to proper names for the body text as suggested by Dustin. Left the irc nicks in the timeline because it corresponds pretty closely with IRC.
